packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,278 @@
+# Terra Application Layout
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
+
+## Props
+
+### `nameConfig`
+
+The `nameConfig` prop allows consuming applications to add their own branding to the ApplicationLayout. The ApplicationLayout will render this content in various ways based on the current breakpoint.
+
+|Key Name|Type|Is Required|Description|
+|---|---|---|---|
+|`accessory`|Element|optional|A React element that will be presented beside the title. This is typically some sort of icon or visual branding.
+|`title`|String|optional|The title of the application. This text will be styled by the ApplicationLayout.|
+
+```jsx
+const myNameConfig = {
+  accessory: <MyApplicationLogo />,
+  title: 'My Application',
+}
+```
+
+### `utilityConfig`
+
+The `utilityConfig` prop allows consuming applications to present an application-level menu directly from the ApplicationLayout. The ApplicationLayout will render this content in various ways based on the current breakpoint.
+
+The API for the `utilityConfig` matches that of the `ApplicationHeaderUtility` and `ApplicationMenuUtility` components from `terra-application-utility`. Please see their respective documentation for more information regarding their full capabilities.
```

Comment:
We should add a link to the [respective documentation](http://engineering.cerner.com/terra-framework/#/site/components/application-utility/index) so it is easier for consumers to get to and figure out more about what to pass to this utilityConfig.

Reply:
I wanted to do that but was unsure which route to link to. With terra-ui up and running with a different url scheme than the other engineering sites, we don't have a single location (aside from github) to point to.

Reply:
I was wondering about this on the overlap of git hub markdown and then site navigation. Could we wrap our site example pages in the embedded content container to allow for redirect in-site to the github url?